src/Header/MainManager.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef Group4_Header_MainManager_H_
+#define Group4_Header_MainManager_H_
+
+#include "HttpListener.h"
+#include "RequestHandler.h"
+#include "SocketProcess.h"
+
+#include<vector>
+
+//will initialize http listener, do multiprocess when listener callback
+class MainManager
+{
+public:
+    //httpListener callback, do fork and new requestHandler to call api, child process add to parents vector for managing.
```

Comment:
Do you want to delete the copy constructor/assignment operator?

Reply:
Done